server/controllers/utils/getFormValues.ts

```diff
@@ -107,7 +107,14 @@ export const getFormValues = ({ errors = {}, unsavedValues = {}, apiValues }: Ge
   }
 
   // these text properties will never use the API value. Subject and details are for notes
-  ;['rescindRequestDetail', 'missingDocumentsDetail', 'subject', 'details'].forEach((key: string) => {
+  ;[
+    'rescindRequestDetail',
+    'missingDocumentsDetail',
+    'subject',
```

Comment:
QQ: Should we favour form specific (unique) property names then - so e.g. rename subject and details here for Notes?  (I'd more naturally favour generic names but I guess this common form handler pushes us towards the specific names - something I hadn't really appreciated).

Reply:
yes. I added a note to loop back and change those two for notes